Administrator Guide/index.md

```diff
@@ -1,74 +1,85 @@
 # Administration Guide
 
-1.  Platform introduction
-
-	*  [About GlusterFS](./GlusterFS Introduction.md)
-	*  [Terminologies](../Quick-Start-Guide/Terminologies.md)
-	
-2.  GlusterFS Architecture and Concepts
-
-	*  [Architecture](../Quick-Start-Guide/Architecture.md)
-	*  [GlusterFS Concepts](./glossary.md)
-	*  [Other Features of GlusterFS](./Did You Know.md)
-	*  [Setting Up Trusted Storage Pools](./Storage Pools.md)
-	*  [Compiling RPMS](./Compiling RPMS.md)
-	*  [Managing the Gluster Service](./Start Stop Daemon.md)
-	
-3.  [Gluster Console Guide](./Console.md)
-
-4.  [POSIX Access Control Lists](./Access Control Lists.md)
-
-5.  [Accessing Data - Setting Up Clients](./Setting Up Clients.md)
-	*  [Handling of users that belong to many groups](./Handling-of-users-with-many-groups.md)
-
-6.  Volume Options
-
-	*  [Setting Up Volumes](./Setting Up Volumes.md)
-	*  [Managing Volumes](./Managing Volumes.md)
-	*  [Building QEMU With gfapi For Debian Based Systems](./Building QEMU With gfapi For Debian Based Systems.md)
-	*  [Modifying .vol files with a filter](./GlusterFS Filter.md)
-	*  [Resolving Peer Rejected](./Resolving Peer Rejected.md)
-	
-7.  [GlusterFS Service Logs and Locations](./Logging.md)
-
-8.  [Brick Naming Conventions](./Brick Naming Conventions.md)
-
-9.  Geo Replication
-
-	*  [Geo Replication](./Geo Replication.md)
-	*  [Distributed Geo Replication](./Distributed Geo Replication.md)
-	
-10.  [Managing Directory Quotas](./Directory Quota.md)
-
-11.  [Managing Snapshots](./Managing Snapshots.md)
-
-12.  [Monitoring Workload](./Monitoring Workload.md)
-
-13.  Data Access With Other Interfaces
-
-	*  [Managing Object Store](./Object Storage.md)
-	*  [Accessing GlusterFS using Cinder Hosts](./GlusterFS Cinder.md)
-	*  [GlusterFS with Keystone](./GlusterFS Keystone Quickstart.md)
-	*  [Hadoop Compatiable Storage with GlusterFS](./Hadoop.md)
-	*  [Install Gluster on Top of ZFS](./Gluster On ZFS.md)
-	*  [Configuring Bareos to store backups on Gluster](./Bareos.md)
-	
-14.  [Securing GlusterFS Communication using SSL](./SSL.md)	
-	
-15.  [Puppet Gluster](./Puppet.md)
-
-16.  [RDMA Transport](./RDMA Transport.md)
-	
-17.  [GlusterFS iSCSI](./GlusterFS iSCSI.md)
-
-18.  [Linux Kernel Tuning](./Linux Kernel Tuning.md)
-
-19.  [Export and Netgroup Authentication](./Export And Netgroup Authentication.md)
-
-20.  [Configuring NFS-Ganesha server](./NFS-Ganesha GlusterFS Intergration.md)
-
-21.  Appendices
-
-	*  [Troubleshooting](./Troubleshooting.md)
-	*  [Network Configurations Techniques](./Network Configurations Techniques.md)
-	*  [Performance Testing](./Performance Testing.md)
+1.  Introduction
+	*  [About GlusterFS](../overview.md)
+	*  [Architecture](./architecture.md)
+
+2.  [Install Guide](../Installation-Guide/install.md)
+
+3.  [Quick Start](../Quick-Start-Guide/Quickstart.md)
+
+4.  [Prerequisites](./prerequisites.md)
+
+5.  [Storage Pool Setup and Scaling](./storage_pool_setup_and_scaling.md)
```

Comment:
This needs retitled. I thought it was a design document until I looked at the text. How about "Trusted Pool Creation and Maintenance" or "Server Peering Administration" or something like that?  Admittedly, after reading the text I completely get this title and am having my own trouble coming up with a title that I think could be understood without context.